### ISSUE TITLE
Mobile Experience :: Disable Touch Zoom On Map When Editing

### ISSUE THREAD

**Issue by SteadyCadence** [Cadasta/cadasta-platform#1127]
### Steps to reproduce the error

With your mobile phone, "Add location" on a project.  Choose a polygon, rectangle or marker from the controls on the left of the map.  

![](https://files.slack.com/files-pri/T0CLFLH1V-F46JDAF0W/screenshot_20170215-101821.png)

#Behavior
When you try to place a place a point on the map, the map zooms in and out instead of allows you to add a point (related to another issue in leaflet). We decided it would be best to disable touch zoom while editing.  The zoom controls are available on the left.

**Comment by laura-barluzzi** [comment]
Hi @dpalomino and @wonderchook, I'm interested in trying out to fix this bug. Could be possible to direct me to the source code of the mobile view at the "Add location" stage? Thanks.

**Comment by oliverroick** [comment]
I'm not sure if we should fix this issue at the moment. We are currently working on a complete rewrite of the maps area; it's probably better to have this sorted once the new implementation is ready.

**Comment by SteadyCadence** [comment]
Since this issue is on hold until the new map is released could we note that shrinking or moving the scale bar may also be required for a good mobile experience?

The scale bar prohibits users from clicking the "point" button on mobile.

**Comment by SteadyCadence** [comment]
I spoke with Frank this morning and a majority of his participants in Indonesia are using their mobile phones for data collection and admin work (review/editing).  I think it is very important to enable users to collect data on the mobile/tablet phones.  In other words, I am trying to bring attention to this issue as it is prohibiting users from collecting date on their mobile phones. :)

**Comment by wonderchook** [comment]
@SteadyCadence appreciate the feedback, we still can't really work on it until the refactoring is done. @clash99 we should think through how all the map interactions with on mobile to make sure it is possible. Then go through the list after the refactor and fix the ones that don't work.

**Comment by dpalomino** [comment]
To be tested after the SMAP work is completed.

**Comment by SteadyCadence** [comment]
FYI @linzjax @dpalomino @wonderchook I am still unable to add locations via mobile phone with smap.

**Comment by SteadyCadence** [comment]
Oh it looks like I can add a polygon (not a rectangle) but we need to disable the touch zoom.

**Comment by dpalomino** [comment]
Thanks @SteadyCadence. 

I've been testing a bit with both a smartphone (mid-range Android device), and a tablet (low-end Android device). Some thoughts: 
- With a tablet (even with a low-end one) experience is far better than with the smartphone (not surprisingly)
- Draw a rectangle doesn't work, but I think we cannot do anything re this. It's just the dragging gesture is not possible. I'd suggest to disable this for mobile browsers. 
- Polygon: it's a bit difficult to draw the geometry (specially when closing the figure trying to click in the last point). Many times you end up adding a new point very close to the last one. Would that be possible to reduce the margin to snap to the last point of the polygon? Again, better with the tablet. 
- Touch zoom (double-tap) is working fine for me... I've had no issues

Maybe we can try to find some time to review user experience with mobile while in Madrid? There are some interesting cases where this would be very useful in the field. 

I will test later on with a fancier tablet to see if there is a big difference.
